.github/ISSUE_TEMPLATE/new-dbt-model.md

```diff
@@ -15,17 +15,17 @@ _(Brief description of the task here.)_
 ## Model attributes
 
 * **Name**: _(What should the model be called? See [Model
- naming](/ccao-data/data-architecture#model-naming) for guidance.)_
+ naming](/ccao-data/data-architecture/tree/master/dbt#model-naming) for guidance.)_
```

Comment:
These links were busted since they pointed to the root of the `data-architecture` repo instead of `data-architecture/dbt`.